dotnet/test/UnitTest.cs

```diff
@@ -0,0 +1,98 @@
+using System;
+using Xunit;
+using System.Text.RegularExpressions;
+
+namespace SecretHubTest

```

Comment:
For now it looks hardcoded since we build and check the library itself instead of the deliverable, which will be the NuGet package. 
The way I can see poeple testing this once the package is available (either locally or online) is by just adding the dependency to the library and installing the package. After that, no hardcoded repo is needed. They can use the test suite in any C# project.
I also have an idea of enabling people to test the package themselves in the repo by changing the script to create the NuGet package and install it automatically. However, I am not sure if that solves the hardcoded repo thing you have in mind, since it may seem similar to what we have now, just that it focuses of testing functionality of the NuGet package, which will be what we intend to release.

Reply:
I am in favor of setting up the test repo in the test suite. But do you think this needs to be done before the release, @jpcoenen?

Reply:
> But do you think this needs to be done before the release, @jpcoenen?

As long as we make sure that it's there for a public release, it should be fine.